packages/kolibri-zip/src/index.js

```diff
@@ -1,67 +1,127 @@
-import { unzip, strFromU8 } from 'fflate';
+import { unzip, strFromU8, strToU8 } from 'fflate';
+import isPlainObject from 'lodash/isPlainObject';
 import loadBinary from './loadBinary';
 import mimetypes from './mimetypes.json';
+import { getAbsoluteFilePath, defaultFilePathMappers } from './fileUtils';
 
-class File {
+class ExtractedFile {
```

Comment:
What extra meaning are we supposed to be understanding from update to "Extracted" File? (I see the JSDocs comments, so maybe this is more just a reflection of me not understanding things about files)

Reply:
It's a little more informative about where the file has come from, but most importantly, it no longer collides with a reserve JS keyword `File` :)

https://developer.mozilla.org/en-US/docs/Web/API/File